scripts/hurl/rs/hrl

```diff
@@ -29,6 +30,7 @@ show_help() {
     echo "    -i <SUBMISSION_ID>    The submissionId to call the history API with (Required for history API)"
     echo "    -v                    Verbose mode"
     echo "    -h                    Display this help and exit"
+    echo "    -p                    Produce a real message, receiver will not be Flexion specific"
```

Comment:
It's not clear to me what this flag is doing given the help message. Is the flag just scrambling the receiver facility details?

Reply:
Description updated to be more concise. Thoughts?

Reply:
The line above also needs updating. I think the update is good, but I'd remove the `Produce a real message` part which can create confusion. I'd also add something explaining the default behavior. It could be something like: `By default, the MSH segment is replaced to contain a Flexion receiver in order to avoid sending messages to partners inadvertently. When using this flag, the MSH segment will not be replaced`

Reply:
I added this to the readme and option description.